go.mod

```diff
@@ -34,21 +34,23 @@ require (
 	golang.org/x/sys v0.7.0
 	golang.org/x/time v0.0.0-20220210224613-90d013bbcef8
 	google.golang.org/grpc v1.51.0
-	k8s.io/api v0.24.6
-	k8s.io/apimachinery v0.24.6
-	k8s.io/apiserver v0.24.6
-	k8s.io/client-go v0.24.6
-	k8s.io/component-base v0.24.6
-	k8s.io/component-helpers v0.24.6
+	k8s.io/api v0.24.16
```

Comment:
is it inevitable to upgrade go mod dependency for k8s-related repo?

Reply:
mainly considered for stability, prefer not to do it if possible

Reply:
This is triggered by the necessary dependency cluster-autoscaler. At the bottom of the go mod, there is a replace directive. Theoretically, the version used is still v0.24.6.